cmd/internal/cmd/compile.go

```diff
@@ -121,8 +121,8 @@ func addCompile(parentCmd *cobra.Command) {
 			}
 
 			fmt.Printf("\nℹ️  Counts\n")
-			for c := range bline.Categories {
-				fmt.Printf(" OSPS-%s: %d controls\n", c, len(bline.Categories[c].Controls))
+			for _, family := range bline.Catalog.ControlFamilies {
+				fmt.Fprintf(os.Stderr, " OSPS-%s: %d controls\n", bline.ControlFamilyIDs[family.Title], len(family.Controls))
```

Comment:
Any reason this is changed to an error out? If I recall this section correctly, it was just intended to give some quick feedback during manual compilation.